src/app/asset-filters/asset-filters.component.pug

```diff
@@ -4,7 +4,7 @@
     .filter-group(*ngIf="availableFilters[filterKey] && isArray(availableFilters[filterKey]) && availableFilters[filterKey].length > 0 && !(availableFilters[filterKey].length === 1 && (availableFilters[filterKey][0].name === '' || availableFilters[filterKey][0].count === 0))")
       span.filter-group__label
         h3 {{ 'FILTERS.' + filterKey | translate }}
-          a#buttonCollectionTypeClear.link--accent.small((keydown.enter)="clearAllFilterGroup(filterKey)", [tabindex]="hasAppliedFilters(filterKey) ? 2 : -1", [ngClass]="{ 'disable': !hasAppliedFilters(filterKey) }", (click)="clearAllFilterGroup(filterKey)", attr.aria-label="Clear, click to clear all selected {{ 'FILTERS.' + filterKey | translate }} filters") Clear
+          a#buttonCollectionTypeClear.link--accent.small(*ngIf="hasAppliedFilters(filterKey)", (keydown.enter)="clearAllFilterGroup(filterKey)", [tabindex]="hasAppliedFilters(filterKey) ? 2 : -1", (click)="clearAllFilterGroup(filterKey)", attr.aria-label="Clear, click to clear all selected {{ 'FILTERS.' + filterKey | translate }} filters") Clear
```

Comment:
Can we extract this?

Reply:
@NazimAli2017 can you elaborate on what exactly you suggest by extracting this?

Reply:
Looks like we could just format it instead, the line is pretty long and difficult to read. Something like this:

```pug
          a#buttonCollectionTypeClear.link--accent.small(
            *ngIf="hasAppliedFilters(filterKey)",
            (keydown.enter)="clearAllFilterGroup(filterKey)",
            [tabindex]="hasAppliedFilters(filterKey) ? 2 : -1",
            (click)="clearAllFilterGroup(filterKey)",
            attr.aria-label="Clear, click to clear all selected {{ 'FILTERS.' + filterKey | translate }} filters."
          ) Clear
```

Reply:
Suggested change made in the recent commit `eab197a`.